### DISCUSSION TITLE
Modify eleven-labs-voices.json file to obtain polish voice

### DISCUSSION THREAD

**Discussion by ldurniat** [SubtitleEdit/subtitleedit#8371] [Q&A]
Hi,

I try to obtain polish voice (for example "Damian PL" with id : S1JKkpuAQNsowB8ZvKRO) by changing SubtitleEditBeta\TextToSpeech\ElevenLabs\eleven-labs-voices.json file.  So far I have modified Rachel voice but with no success. 
Below current entry for new voice :   

```
{
            "voice_id": "S1JKkpuAQNsowB8ZvKRO",
            "name": "Rachel",
            "labels": {
                "accent": "standard",
                "descriptive": "professional",
                "language": "pl",
                "age": "middle_aged",
                "gender": "male",
                "use case": "narrative_story"
            },
            "model_id": "eleven_multilingual_v2",
        },

```
However I have noticed that SE call ElevelLabs API with option model_id : "eleven_monolingual_v1"  but on web page "Damian PL" voice uses model_id : "eleven_multilingual_v2". How can I set model_id option in eleven-labs-voices.json ? 

Have a nice day:)

![voice_id](https://github.com/SubtitleEdit/subtitleedit/assets/22639656/7cb0afe9-24c2-4e77-a0c3-b8b92c9aeb27)

![voice_id2](https://github.com/SubtitleEdit/subtitleedit/assets/22639656/3d163ebc-5b88-4ea9-9f78-a720d9fc2a70)

**Comment by niksedk** [comment]
Latest beta uses `eleven_multilingual_v2` 

You can also refresh voices (do use your api key) via right-click on voices combo box
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/c94daf5b-da68-4d78-844c-0fc87d5cf532)

https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip

**Comment by laszlolaszlo** [comment]
It should be fine for me to use somewhat Eleven Turbo v2.5 because Hungarian language only in this modell. Thank you!
[Eleven Labs doc](https://help.elevenlabs.io/hc/en-us/articles/17883183930129-What-models-do-you-offer-and-what-is-the-difference-between-them)
